<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hero</class>
 <widget class="QDialog" name="Hero">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>250</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 20pt &quot;Sans Serif Collection&quot;;</string>
   </property>
   <property name="text">
    <string>Your Team</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>40</y>
     <width>250</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Sans Serif Collection&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Opponent Team</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="VS">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 30pt &quot;Sans Serif Collection&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>VS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>289</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QComboBox" name="team1_hero2_select_menu"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="team1_hero4_select_menu"/>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="team1_hero3_select_menu"/>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="team1_hero1_select_menu"/>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="team1_hero5_select_menu"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>150</y>
     <width>289</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QComboBox" name="team2_hero2_select_menu"/>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="team2_hero1_select_menu"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="team2_hero4_select_menu"/>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="team2_hero3_select_menu"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="team2_hero5_select_menu"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Select_hero">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="predicted_accuracy_of_all_heroes">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>470</y>
     <width>631</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="hero1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hero2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hero3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="hero4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hero5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
